visualization/app/codeCharta/ui/fileChooser/fileChooser.e2e.ts

```diff
@@ -27,9 +30,14 @@ describe("FileChooser", () => {
 		expect(await page.$eval("#loading-gif-map", el => el["className"])).toContain("ng-hide")
 	})
 
-	it("should not load non json file", async () => {
+	it("should open an invalid file, close the dialog and open a valid file", async () => {
 		await fileChooser.openFile("./app/codeCharta/assets/logo.png")
+		expect(await dialogError.getMessage()).toEqual(" file is empty or invalid")
 
-		expect(await filePanel.getSelectedName()).toEqual("sample1.cc.json")
+		await dialogError.clickOk()
+
+		await FileChooserPageObject.selectFile("./app/codeCharta/assets/sample3.cc.json")
```

Comment:
Good catch that the functions on `FileChooserPageObject` are actually static. I would suggest to change the other ones as well, while being on it?

Reply:
True. I'd do this in a separate PR though. We gotta update a lot of files and a template.

Reply:
I opened an issue  #1142